### PR TITLE
🐛✅Fix tickevent tests for Safari

### LINKED ISSUE #21062: Unmute some performance tests
Muted by #21061

@ericandrewlewis could you pls take a look?

### PR DESCRIPTION
**Pull request by ericandrewlewis**
Fixes #21062

### CODE REVIEW

File: src/service/performance-impl.js [hunk 1]
```diff
@@ -230,7 +230,15 @@ export class Performance {
...
     });
-    observer.observe({entryTypes: entryTypesToObserve});
+
+    // Wrap observer.observe() in a try statement for testing, because
+    // Webkit throws an error if the entry types to observe are not natively
+    // supported.
+    try {
+      observer.observe({entryTypes: entryTypesToObserve});
+    } catch (err) {
+      console/*OK*/.warn(err);
```
Comment (lannka) [inline src/service/performance-impl.js]: dev().warn()

File: src/service/performance-impl.js [hunk 2]
```diff
@@ -230,7 +230,15 @@ export class Performance {
       list.getEntries().forEach(processEntry);
       this.flush();
     });
-    observer.observe({entryTypes: entryTypesToObserve});
+
+    // Wrap observer.observe() in a try statement for testing, because
+    // Webkit throws an error if the entry types to observe are not natively
```
Comment (lannka) [inline src/service/performance-impl.js]: so we're OK to not measure safari traffic?

File: src/service/performance-impl.js (hunk 1 above)
Comment (ericandrewlewis) [reply src/service/performance-impl.js]: Done

File: src/service/performance-impl.js (hunk 2 above)
Comment (ericandrewlewis) [reply src/service/performance-impl.js]: We won't be able to read metrics from Performance APIs that Webkit hasn't implemented yet like EventTiming and LayoutJank, but when they do we will start getting numbers.

**Comment by lannka** [comment]
Pls fix the lint

**Comment by ericandrewlewis** [comment]
Lint fixed @lannka

**Pull request merged**